metricbeat/helper/prometheus/ptest/ptest.go

```diff
@@ -0,0 +1,106 @@
+package ptest
```

Comment:
I wonder if we should call these packages always `testing` as we do in some other places or if this is always and issue because there is a `testing` package.

Note: Just for discussion, no change in the PR needed.

Reply:
I had the same thought, but as this is always used along with the Go `testing` package clash is guaranteed.

This already happens with the `mb/testing` package, we end up renaming it on import to `mbtest`, with the annoyance of breaking automatic imports.

Reply:
There is another example in the latest changes on RabbitMQ module, all modules end up being renamed, but I think this is not so bad (https://github.com/elastic/beats/pull/7106/files#diff-7d67ada6eac5ae4109019e586c93d3a1R8)

I'm slightly in favour of having the same name for all test modules, though not sure if `testing` is the best option as it collides with the standard library.

Reply:
For the module testing packages I think all must have the same name or structure, as we need to somehow modify the `imports` script to skip them. In general what i worry about directories in module is that they can always conflict with metricset names.